src/Components/Filter/Filter.js

```diff
@@ -0,0 +1,208 @@
+import React from "react";
+import { categories } from "../../backend/db/categories";
+import { products } from "../../backend/db/products";
+import { useProduct } from "../../contexts/ProductProvider";
+
+export const getSortedData = (state, productList) => {
+  if (state.sortBy === "PRICE_HIGH_TO_LOW") {
+    return [...productList].sort((a, b) => b.price - a.price);
+  }
+  if (state.sortBy === "PRICE_LOW_TO_HIGH") {
+    return [...productList].sort((a, b) => a.price - b.price);
+  }
+  return productList;
+};
```

Comment:
Suggestion: you can put this functions in utils folder

Reply:
I already added it in the next PR.